packages/squiggle-lang/src/dist/SymbolicDist.ts

```diff
@@ -1070,6 +1071,119 @@ export class PointMass extends SymbolicDist {
   }
 }
 
+export class Binomial extends SymbolicDist {
+  constructor(
+    public n: number,
+    public p: number
+  ) {
+    super();
+  }
+  toString() {
+    return `Binomial(${this.n},{${this.p}})`;
+  }
+  static make(n: number, p: number): result<Binomial, string> {
+    if (!Number.isInteger(n) || n < 0) {
+      return Result.Err(
+        "The number of trials (n) must be a non-negative integer."
+      );
+    }
+    if (p < 0 || p > 1) {
+      return Result.Err("Binomial p must be between 0 and 1");
+    }
+    return Ok(new Binomial(n, p));
+  }
+
+  simplePdf(x: number) {
+    return jstat.binomial.pdf(x, this.n, this.p);
+  }
+
+  cdf(k: number) {
+    return jstat.binomial.cdf(k, this.n, this.p);
+  }
+
+  inv(p: number): number {
+    throw new Error("Binomial inv not implemented");
```

Comment:
I see that these dists and methods really don't get used now; `Danger.binomialDist` gets immediately transformed to the sampleset, and there's no `Sym.binomial` yet.

Can you add a comment about that here if you don't want to implement these? For a real symbolic binomial dist, having `inv` and `toPointSetDist` is critical, it won't be rendered as a chart otherwise.